doc/apiary.apib

```diff
@@ -0,0 +1,754 @@
+FORMAT: 1A
+HOST: http://localhost:8000
```

Comment:
The default port for the device manager is 5000, this host is set considering the service by itself or with the API Gateway up front to expose a different port?

Reply:
In fact, this should make no difference. This HOST tells apiary that the "production" server is accessible from this address. As we have no such thing for now, I think this might better be removed.